index.html

```diff
@@ -27,110 +28,373 @@
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
+ 
     // SCALES
 
+
+    chart.x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.Reading_proficient/100; })]) //finds highest value of reading scores
+      .range([0, width])
+      .nice();
+
+
+    chart.y = d3.scale.linear()
+     // .domain(d3.extent(data, function (d) { return d.Math_proficient/100; })) 
+      .domain([0, d3.max(data, function (d) { return d.Math_proficient/100; })])
+      .range([height, 0])
+      .nice();
+
     // AXES
 
-    // EVENT HANDLERS
+    var formatPercentage = d3.format("1%");
+    var formatPercentage_y = function(d) { if (d > 0) {
+      return formatPercentage(d); }
+    };
 
-    d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
-      update();
-    });
+    var xAxis = d3.svg.axis()
+        .scale(chart.x)
+        .ticks(10)
+        .orient("top")
+        .tickSize(height) 
+        .tickFormat(formatPercentage);
+
+    var yAxis = d3.svg.axis()
+        .scale(chart.y)
+        .ticks(10)
+        .orient("left")
+     //   .outerTickSize(0)
+        .tickSize(-width, 0, 0) 
+        .tickFormat(formatPercentage_y);
+
+    var gx = chart.svg.append("g")
+        .attr("class", "x axis")
+        .attr("transform", "translate(0," + (height) + ")")
+        .call(xAxis)
+    gx.selectAll("g").filter(function(d) { return d;}) //need to use filter since true values do not return 0
+        .classed("minor", true);
+   //   .attr("transform")
+    gx.selectAll("text")
+       .attr("transform", "translate(0," + (height + margin.bottom/2) + ")")
+    gx.append("text")
+        .attr("text-anchor", "middle")
+        .attr("transform", "translate(" + (width /2) + " ," + (margin.bottom*.85) + ")")
+        .style("text-anchor", "middle")
+        .text("Reading proficiency");
+
+ 
+
+    var gy= chart.svg.append("g")
+        .attr("class", "y axis")
+     //   .attr("transform", "translate(" + ",0)")
+        .call(yAxis)
+    gy.selectAll("g").filter(function(d) { return d;})
+        .classed("minor", true);
+    gy.selectAll("text") 
+       .attr("x", -1)
+       .attr("dy", 0);
+    gy.append("text")
+       .attr("transform", "rotate(-90)")
+       .attr("y", -42)
+       .attr("x", margin.top - (height/2))
+       .attr("dy", ".71em")
+       .style("text-anchor", "middle")
+       .text("Math proficiency");
+
+    //TOOLTIP
+
+    chart.tooltip = d3.tip()
+        .attr('class', 'tooltip')
+        .offset([-6, 0])
+        .html(function(d) { return d.length + '% of schools reported that ' + formatPercentage(d.x) + ' to ' + formatPercentage(d.x + .1) + '<br/>' +' of ' + '<u>' + options.categoryRange + '</u>' +' students met basic proficiency levels.' });
+
```

Comment:
Very nice! Both Chris and I were impressed with the tooltip. Especially on including not just the percentage, but also the histogram bin and the pertinent subgroup.